pkg/vk8s/container.go

```diff
@@ -43,6 +54,15 @@ func getNamedContainer(cnts []corev1.Container, cntName string) *corev1.Containe
 	return nil
 }
 
+func getNamedContainerStatus(cntStatuses []corev1.ContainerStatus, cntName string) *corev1.ContainerStatus {
```

Comment:
A similar function `findContainerStatus`, was added in main, but it wasn't in vnext yet. I have just merged main into vnext. Can you merge up and use the new function?

Reply:
I saw that function but I would like it to be `findContainerStatus(cntStatuses []corev1.ContainerStatus, cntName string)` to have some flexibility because scrutinitize init container status belongs to `pod.Status.InitContainerStatuses` so I can't use `findContainerStatus` as it is.

Reply:
I'm okay with changing the function signature.